app/views/routes/_form.html.slim

```diff
@@ -0,0 +1,16 @@
+= form_with model: route, local: true do |f|
+  = render partial: 'shared/error', locals: { model: route }
+  p
+    = f.label :name
+    br
+    = f.text_field :name
+  p
+    = f.label :length_of_time
+    br
+    = f.text_field :length_of_time
+  p
+    = f.label :truck
+    br
+    = f.collection_select :truck_id, Truck.all, :id, :license_plate, include_blank: 'Select a Truck'
```

Comment:
Well, it actually arguable and could bring a holy war here, but some sources tell to avoid usage of direct Active Record calls in the view.
So you could try to avoid using `Truck.all` here.
Try to extract it to the helper method or service or.... just forget about it :)

Reply:
yeah ideally a helper or something is the way to go, but we can do it as a refactor exercise a bit later if we have time

Reply:
> yeah ideally a helper or something is the way to go, but we can do it as a refactor exercise a bit later if we have time

ok so i dont add the helper first

Reply:
Could pretty easily define in your controller:
```
@trucks = Truck.all
```
then pass it into this  form partial with
```
render partial: 'form', locals: { trucks: @trucks }
```
certainly don't need a helper